src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -272,13 +272,17 @@ private void processExternSymbols() {
       String parentPath = getNamespace(symbol.getName());
       boolean isDefault = isDefaultExport(symbol);
 
+      // skip foo.bar.baz if:
+      // * foo.bar is namespace
+      // * foo.bar is class-like and baz is a static field.
+      // * foo is a class-like and foo.bar is a static field.
```

Comment:
maybe move the comments next to the if statements? Might also be interesting to have a note like "these are handled by xxx.", or at least "elsewhere".